solidity/contracts/CronosGravity.sol

```diff
@@ -49,6 +56,9 @@ contract CronosGravity is Gravity, AccessControl, Pausable, Ownable {
         _setupRole(RELAYER_ADMIN, relayerAdmin);
         _setRoleAdmin(RELAYER, RELAYER_ADMIN);
         _setRoleAdmin(RELAYER_ADMIN, RELAYER_ADMIN);
+
+        // Set to approximately one day
+        migrationPeriod = 6500;
```

Comment:
1 day is too short, I vote for 3 days (taking into consideration of weekends in case those not paying attention during that time).
DeFi protocol has minimum of 2 days, with some doing  3 days or 5 days.
https://compound.finance/docs/governance#timelock
https://governance.aave.com/t/arc-activation-of-aave-protocol-governance-v2/1717
https://boardroom.io/uniswap/overview

Reply:
ok, changed to 3days